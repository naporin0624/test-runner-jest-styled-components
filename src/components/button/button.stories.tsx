import { ComponentPropsWithoutRef } from "react";
import { Button } from ".";

export default {
  title: "Button",
  component: Button,
};

type Props = ComponentPropsWithoutRef<typeof Button>;
const DefaultStory = (args: Props) => {
  return <Button {...args}>default button</Button>;
};

export const Default = DefaultStory.bind({});
