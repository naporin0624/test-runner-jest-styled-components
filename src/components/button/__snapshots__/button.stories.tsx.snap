// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Default smoke-test 1`] = `
<button class="sc-beySPh fvFDzo">
  default button
</button>
`;
